docs/60_PACKAGE.md

```diff
@@ -51,6 +51,35 @@ directory.  Defaults to the docroot.
 packages directory. This can be overridden by calling grunt package with the
 `--name` parameter.
 
+**archive**: Used to enable the `compress` option via the configuration instead
+of the command line. Defaults to false.
+
+**rsync**: If set to true, rsync will be used instead of file copying for
+improved performance. This automatically enables the `compress` option and will
+write the archive to the package destination.
+
+**vmData**: This path is used as the intermediate generation directory for the
+package when using the `rsync` option. By selecting a volume mounted in your
+VM `/data` area, this can significantly improve performance. The archive is
+still moved to the normal package destination when complete.  Defaults to
+`build/vm_data`.
+
+## Performance
+
+When running `grunt package` within a docker container, the default file
```

Comment:
I think I covered this above, but the whole point of this PR was performance.  Switching from copy to rsync gives a small performance improvement.  The majority of the improvement comes from using the vm_data.

Yes, it's really about NFS and not docker, so the docs can be wordsmithed a bit.

And yes, this vm_data is definitely something we want to incorporate into other areas of gdt.  The package command was just the big target because it was taking 40 minutes on a project.